src/components/dialog/dialog.component.js

```diff
@@ -273,6 +289,12 @@ Dialog.propTypes = {
   bespokeFocusTrap: PropTypes.func,
   /** The ARIA role to be applied to the Dialog container */
   role: PropTypes.string,
+  /** Padding to be set on the Dialog content */
+  contentPadding: PropTypes.shape({
+    p: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6, 7, 8]),
```

Comment:
Did it come from UX to lock these props down to 8px multipliers only? As a user I would expect these to work in the same way as all our other padding props, where we can pass in any valid CSS string

Reply:
Yeah I spoke with @harpalsingh about this and we agreed for now this basic interface is all that we'll need, if consumers request a more flexible one we can open it up at that point. The issue is the need to calculate the values for the form margins, if we allow all the variations (strings, px/ %/ em and so on) it becomes quite complex and messy to pull the values out.

Reply:
Ok makes sense 👍🏻